recursio-client/Characters/PlayerGhost.tscn

```diff
@@ -12,7 +12,7 @@ script = ExtResource( 4 )
 color_scheme = "player_ghost"
 
 [node name="VisibilityLight" parent="KinematicBody" index="3" instance=ExtResource( 2 )]
-visible = false
+transform = Transform( 1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0.368, -0.741 )
```

Comment:
Should the light be over the player? `0.368` means it's quite a bit higher right?

Reply:
I just copied the values from the player light, should be fine though, that's how it worked for the player the whole time.